corehq/apps/reports/standard/export.py

```diff
@@ -280,6 +280,7 @@ def _sortkey(form):
                 if group.form_exports],
             report_slug=self.slug,
             property_hash=self.properties(size_hash),
+            has_media=self.request.project.has_media(),
```

Comment:
Ah, I think you're right on that, it goes through this which seemed like it was for multimedia submissions not just multimedia in the app itself, but I guess not.. https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqmedia/models.py#L501-508